plugins/chat/assets/javascripts/discourse/components/chat-message-info.js

```diff
@@ -64,7 +34,36 @@ export default class ChatMessageInfo extends Component {
     if (user.groupModerator) {
       classes.push("is-category-moderator");
     }
-
+    if (user.new_user) {
+      classes.push("is-new-user");
+    }
+    if (user?.primary_group_name?.length) {
```

Comment:
nit but why is `.length` needed, can'y we just do `if user?.primary_group_name`...also no need to do `user?` here since above we already don't do safe navigation and early return

Reply:
yes the `user?` is a mistake :+1:

Concerning length, I copied behavior from core too. I Imagine we sometimes store an empty string here.